airflow-core/src/airflow/serialization/serde.py

```diff
@@ -365,30 +365,29 @@ def _register():
     _stringifiers.clear()
 
     with Stats.timer("serde.load_serializers") as timer:
-        for _, name, _ in iter_namespace(airflow.serialization.serializers):
-            name = import_module(name)
-            for s in getattr(name, "serializers", ()):
-                if not isinstance(s, str):
-                    s = qualname(s)
-                if s in _serializers and _serializers[s] != name:
-                    raise AttributeError(f"duplicate {s} for serialization in {name} and {_serializers[s]}")
+        for _, module_name, _ in iter_namespace(airflow.serialization.serializers):
+            module = import_module(module_name)
+            for serializers in getattr(module, "serializers", ()):
+                s = serializers if isinstance(serializers, str) else qualname(serializers)
+                if s in _serializers and _serializers[s] != module:
+                    raise AttributeError(f"duplicate {s} for serialization in {module} and {_serializers[s]}")
                 log.debug("registering %s for serialization", s)
-                _serializers[s] = name
-            for d in getattr(name, "deserializers", ()):
-                if not isinstance(d, str):
-                    d = qualname(d)
-                if d in _deserializers and _deserializers[d] != name:
-                    raise AttributeError(f"duplicate {d} for deserialization in {name} and {_serializers[d]}")
+                _serializers[s] = module
+            for deserializers in getattr(module, "deserializers", ()):
+                d = deserializers if isinstance(deserializers, str) else qualname(deserializers)
+                if d in _deserializers and _deserializers[d] != module:
+                    raise AttributeError(
+                        f"duplicate {d} for deserialization in {module} and {_deserializers[d]}"
+                    )
                 log.debug("registering %s for deserialization", d)
-                _deserializers[d] = name
+                _deserializers[d] = module
                 _extra_allowed.add(d)
-            for c in getattr(name, "stringifiers", ()):
-                if not isinstance(c, str):
-                    c = qualname(c)
-                if c in _deserializers and _deserializers[c] != name:
-                    raise AttributeError(f"duplicate {c} for stringifiers in {name} and {_stringifiers[c]}")
+            for stringifiers in getattr(module, "stringifiers", ()):
```

Comment:
Probably not something we need to do it this PR. but we maybe can duplicate these few for loop 🤔